Automattic/hostmgr
# Fix sync

Fixes sync with better conditional checking.

**To Test**
- Check out this PR, and run `swift run hostmgr sync authorized_keys`. Ensure that it runs correctly.
